src/main/java/core/basesyntax/service/operation/BalanceOperationHandlerImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandlerImpl implements OperationHandler {
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
```
the same for other OperationHandler implementations